src/app/features/test-records/views/test-result-summary/test-result-summary.component.html

```diff
@@ -0,0 +1,134 @@
+  <ng-container *ngIf="techRecord$ | async as techRecord">
+    <app-vehicle-header
+      [testResult]="testResult"
+      [techRecord]="techRecord"
+      [isEditing]="false"
+      [testNumber]="(routerParam('testNumber') | async) || ''"
+    >
+      <ng-container *appRoleRequired="Roles.TestResultAmend">
+        <a
+          *ngIf="isTestTypeGroupEditable$ | async; else notSupported"
+          [routerLink]="['amend-test']"
+          role="button"
+          draggable="false"
+          class="govuk-button govuk-button--secondary"
+          data-module="govuk-button"
+          >Amend this test
+        </a>
+      </ng-container>
+    </app-vehicle-header>
+
+    <article>
+      <div class="vehicle_heading govuk-!-margin-bottom-6">
+        <h2 class="govuk-heading-l">
+          <span>{{ techRecord?.bodyMake + '-' + techRecord?.bodyModel | defaultNullOrEmpty }}</span>
+
+          <app-icon [icon]="techRecord?.vehicleType || ''"></app-icon>
+        </h2>
+      </div>
+      <h3 class="govuk-heading-m">Vehicle overview</h3>
+      <div class="govuk-summary-list">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">VRM</dt>
+          <dd id="test-testTypeId" class="govuk-summary-list__value">
+            <app-number-plate>
+              {{ testResult?.vrm | defaultNullOrEmpty }}
+            </app-number-plate>
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">VIN</dt>
+          <dd id="test-testTypeId" class="govuk-summary-list__value">
+            {{ testResult?.vin | defaultNullOrEmpty }}
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Vehicle type</dt>
+          <dd id="test-testTypeId" class="govuk-summary-list__value">
+            {{ techRecord?.vehicleType | defaultNullOrEmpty | uppercase }}
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Country of registration</dt>
+          <dd id="test-testTypeId" class="govuk-summary-list__value">
+            {{ testResult?.countryOfRegistration | defaultNullOrEmpty | uppercase }}
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">EU vehicle ategory</dt>
+          <dd id="test-testTypeId" class="govuk-summary-list__value">
+            {{ testResult?.euVehicleCategory | defaultNullOrEmpty }}
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Odometer reading</dt>
+          <dd id="test-testTypeId" class="govuk-summary-list__value">
+            {{ testResult?.odometerReading | number: '2.' | defaultNullOrEmpty }}
+            {{ testResult?.odometerReadingUnits === 'kilometres' ? 'km' : ('mi' | defaultNullOrEmpty) }}
```

Comment:
You don't need the `defaultNullOrEmpty` pipe here.

Reply:
I do but that expression was poop so changed it. Thanks :)

Reply:
How can the string `'mi'` ever be `null` or `''`?